### PR TITLE
Add french translation

### PR DESCRIPTION
**Pull request by azizdiakite**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] My work includes tests or is validated by existing tests.

## Summary
https://openmrs.atlassian.net/browse/O3-4041

### CODE REVIEW

**Review by reagan-meant — APPROVED** [review]
LGTM. Thanks @azizdiakite

File: translations/fr.json [hunk 1]
```diff
@@ -1,61 +1,67 @@
...
+  "updateEncounter": "Mettre à jour les résultats d'analyse",
+  "urgency": "Urgence",
+  "view": "Voir",
+  "worklist": "Analyses en cours",
+  "noLabRequestsFoundC": "Aucune demande d'analyse trouvée",
+  "checkFilters": "Veuillez vérifier les filtres ci-dessus et réessayer",
+  "startDate":"Date de début",
+  "endDate":"Date de fin",
+  "totalOrders":"Total des demandes",
+  "itemsPerPage": "Éléments par page"
```
Comment (denniskigen) [inline translations/fr.json:66]: Thank you for working on this, @azizdiakite! So, we don't modify non-`en` translation files by hand. Instead, we use the Transifex integration for that. These strings ought to be translated via the Transifex UI. Automated PRs will then provide the updated strings back to the repo ([similar to this one](https://github.com/openmrs/openmrs-esm-laboratory-app/pull/98)).

Read more about this [here](https://om.rs/translate).

File: src/index.ts [hunk 2]
```diff
@@ -18,7 +18,7 @@ export const laboratoryDashboardLink = getSyncLifecycle(
   createHomeDashboardLink({
     name: 'laboratory',
     slot: 'laboratory-dashboard-slot',
```
Comment (denniskigen) [inline src/index.ts:20]: This is the canonical way to translate the dashboard link text:

```suggestion
  // t('laboratory', 'Laboratory')
  createHomeDashboardLink({
    name: 'laboratory',
    slot: 'laboratory-dashboard-slot',
    title: 'Laboratory'
```

File: src/components/create-dashboard-link.component.tsx [hunk 3]
```diff
@@ -24,7 +26,7 @@ function DashboardExtension({ dashboardLinkConfig }: { dashboardLinkConfig: Dash
       to={`${spaBasePath}/${name}`}
       className={`cds--side-nav__link ${navLink.match(name) && 'active-left-nav-link'}`}
     >
-      {title}
+      {t(title, 'Laboratory')}
```
Comment (denniskigen) [inline src/components/create-dashboard-link.component.tsx:29]:
```suggestion
      {t(title)}
```

**Review by denniskigen — CHANGES_REQUESTED** [review]

**Pull request closed without merging**